src/components/Link/Link.tsx

```diff
@@ -40,16 +40,17 @@ function linkClasses<T>(
 ): string | undefined {
   const unstyled = variant === 'unstyled'
   const isExternalLink = variant === 'external'
+  const isNavLink = variant === 'nav'
 
   return unstyled
     ? className
```

Comment:
The way the [classnames](https://github.com/JedWatson/classnames#usage/) package works is it already conditionally includes the css. This means  we don't need to add another ternary here.  Check out the linked docs if you've never used this package before, its pretty convenient.  

See if you could get this still passing tests but following a pattern that's more similar to this pseudocodey example:
``` javascript
return unstyled?
    className // if its unstyled, only use classes passed in by props
     :  
   classnames( // otherwise use...
       'usa-link', // any classes required for component
       { 'random-class': true},  // and  any conditionally included CSS
       classname  // and also add the classes passed in by props
    )
```

We add `className` last because we always want the styles that come in through props to be applied on top of other styles. There are several examples of this throughout react-uswds codebase to check out too if you want.